src/Extracting/ParsesValidationRules.php

```diff
@@ -549,7 +549,7 @@ protected function processRule($rule, $ruleArguments, array &$parameterData, arr
                     $parameterData['nullable'] = true;
                     break;
                 case 'exists':
-                    $parameterData['description'] .= " The <code>{$ruleArguments[1]}</code> of an existing record in the {$ruleArguments[0]} table.";
+                    $parameterData['description'] .= " The <code>{$ruleArguments[1]}</code> value must exist.";
```

Comment:
```suggestion
                    $parameterData['description'] .= " The <code>{$ruleArguments[1]}</code> of an existing record";
```

I'd rather go with this.